commit e384f94920505f64357ffd6f331859ed71cb8281
Author: Raphael Kubo da Costa <rakuco@FreeBSD.org>
Date:   Wed Apr 13 20:08:14 2016 +0200

    libkipiplugins: Add dependency on the kipiplugins-gitversion.
    
    kpversion.cpp needs gitversion.h, which is generated by the
    kipiplugins-gitversion target.
    
    If the dependency is not specified, a parallel build can break as
    kpversion.cpp may be built before gitversion.h is generated.

[adapted to this version of kipi-plugins]
--- common/libkipiplugins/CMakeLists.txt
+++ common/libkipiplugins/CMakeLists.txt
@@ -34,6 +34,7 @@
   )
 
 KDE4_ADD_LIBRARY(kipiplugins SHARED ${kipiplugins_LIB_SRCS})
+add_dependencies(kipiplugins kipiplugins-gitversion)
 
 target_link_libraries(kipiplugins
                       ${QT_QTGUI_LIBRARY}
